### DISCUSSION TITLE
Openwrt怎么更新alist

### DISCUSSION THREAD

**Discussion by stringliao** [AlistGo/alist#4879] [Q&A]
最新alist只有一个gz文件，Openwrt里面怎么来更新，有大侠可以指导一下吗？

**Comment by chengnan049** [comment]
sh -c "$(curl -ksS https://raw.githubusercontent.com/sbwml/luci-app-alist/master/install.sh)"

**Comment by a447948814** [comment]
> sh -c "$(curl -ksS https://raw.githubusercontent.com/sbwml/luci-app-alist/master/install.sh)"

不行 失效了

**Reply by a447948814** [reply]
https://github.com/sbwml/luci-app-alist
